src/Screens/Home/HomeScreen.js

```diff
@@ -1,15 +1,52 @@
-import { View, Text, Button } from "react-native";
+import { Text, ScrollView } from "react-native";
 import React from "react";
+import styles, { Color } from "../../Shared/styles.js";
+import HomeScreenButton from "../../Shared/Components/HomeScreenButton.js";
 
 const HomeScreen = ({ navigation }) => {
   return (
-    <View>
-      {/*  button is added to test navigation, component button will replace this */}
-      <Button
-        title="Legislation"
+    <ScrollView style={styles.homeScreenBackground}>
+      <Text style={styles.homeScreenHeader}>What are you looking for?</Text>
+      <Text style={styles.homeScreenSubHeader}>
+        Explore supporting information and resources
+      </Text>
+
+      {/* Legislation Button */}
+
+      <HomeScreenButton
```

Comment:
Buttons need to map from a create JSON, this was a admendement discussed during one of the sprint meetints